src/components/SelectParatime.tsx

```diff
@@ -0,0 +1,67 @@
+import { KeyboardArrowDown } from '@mui/icons-material';
+import { MenuItem, Select } from '@mui/material';
+
+
+interface SelectParatimeProps {
+  selectedParatime: string;
+  handleParatimeChange: (value: string) => void;
+}
+
+const SelectParatime: React.FC<SelectParatimeProps> = ({ selectedParatime, handleParatimeChange }) => {
```

Comment:
The ParaTime selection should be part of the filter I think. 90% of examples will be Sapphire-only, so there is no need to show the ParaTime filter on the first page right next to the Search field.